tracing-subscriber/src/fmt/fmt_layer.rs

```diff
@@ -1041,9 +1041,9 @@ where
 {
     /// Visits every span in the current context with a closure.
     ///
-    /// The provided closure will be called first with the current span,
-    /// and then with that span's parent, and then that span's parent,
-    /// and so on until a root span is reached.
+    /// The provided closure will be called first with the root span,
+    /// and then traverse through all the children until it reaches 
```

Comment:
I think that "all the children" is potentially misleading as one could take that to mean the children of the root span.

I can't really think of how to make the wording clearer, for reference this is how `SpanRef::scope` documents this so maybe use something similar (but reverse and with callback)?

```
    /// Returns an iterator over all parents of this span, starting with this span,
    /// ordered from leaf to root.
```

Reply:
Personally, I would prefer the last one.

The reason being when we talk about "parents of a span", it should be obvious that we're talking about the parent, its parent, and so on (assuming the reader knows there is always at most one parent), while with children, this logical jump is not possible.

Something like "ancestors" might be even better, but I don't think that terminology like that is used.

But it may be just my bias so feel free to get a second opinion.